webapp/src/features/event/EventRoot.tsx

```diff
@@ -65,13 +66,33 @@ const EventRoot = () => {
         setEventScratchpad(event);
     }
 
+    const publishEvent = async () => {
+        if(hasChanges) {
+            alert("Please save changes before publishing this event.\nUwU");
```

Comment:
We should use the bootstrap toast instead of the standard alert.  This ain't 2002